### DISCUSSION TITLE
Firebase email authentication ionic angular

### DISCUSSION THREAD

**Discussion by Muhammad-Khuram** [ionic-team/capacitor#7311] [Q&A]
How I make that when user type email or password for sign up on front-end and click sign up button then firebase send OTP code to user email account.

**Comment by izyuumi** [comment]
This page from Firebase documents how to use Firebase Authentication with email link. https://firebase.google.com/docs/auth/web/email-link-auth#send_an_authentication_link_to_the_users_email_address
